packages/cards/src/actions/submit/task-fetch.ts

```diff
@@ -2,6 +2,13 @@ import { ISubmitAction, MSTeamsData, SubmitAction, SubmitActionOptions } from '.
 
 export type TaskFetchActionOptions = SubmitActionOptions & { data: MSTeamsData<ITaskFetchData> };
 
+export type ValidTaskFetchDataValues = {
```

Comment:
This is an opportunistic fix that I'm happy to revert for now. I found that `type` being supplied to data was causing it to not send invokes back. Happy to do more testing before checking this in.

Reply:
no this seems fine, only nit is that unless there is an `InvalidTaskFetchDataValues` I think we can just call this type `TaskFetchDataValues`, since it can be implied as valid data if it makes our types